core/src/main/java/com/exadel/aem/toolkit/core/lists/models/internal/ListItemModel.java

```diff
@@ -99,6 +92,36 @@ public String getItemResType() {
         return properties;
     }
 
+    /**
+     * Retrieves the path to a custom renderer for the 'preview' section of the List item.
```

Comment:
(cosmetic) The latest trend)) around the project is no use `{@code}` sections instead of single quote marks in Javadoc

Reply:
Done